features/step_definitions/node-update-notification-steps.js

```diff
@@ -0,0 +1,68 @@
+import { When, Then } from 'cucumber';
+import { expect } from 'chai';
+
+const NODE_UPDATE_COMPONENT = '.NodeUpdateNotification_component';
+const TITLE_BAR = '.NodeUpdateNotification_titleBar';
+const TOGGLE_BUTTON = '.NodeUpdateNotification_toggleButton';
+const UPDATE_MESSAGE = '.NodeUpdateNotification_message';
+const ACTIONS = '.NodeUpdateNotification_actions';
+const ACCEPT_BTN = '.NodeUpdateNotification_acceptButton';
+const DENY_BTN = '.NodeUpdateNotification_denyButton';
+
+When(/^I make a node update available$/, async function () {
+  await this.client.executeAsync((nextVersion, done) => {
+    daedalus.api.ada.setNextUpdate(nextVersion)
+      .then(() => daedalus.stores.NodeUpdateStore.refreshNextUpdate())
+      .then(done)
+      .catch((error) => done(error));
+  }, { version: 50 });
```

Comment:
@MarcusHurney please extract this version value to a constant which we can use in later steps where you check if the version shown in the notification matches this expected one.